docs/serving/samples/cloudevents/cloudevents-dotnet/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: cloudevents-dotnet
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - image: <image>
+          ports:
+          - containerPort: 80
```

Comment:
How hard would it be to make this 8080?

Reply:
Not hard. Pushed a change to listen on the default 8080 port.